app/src/App/utils.tsx

```diff
@@ -34,6 +35,10 @@ export const teamAssignmentProps = {
   defaultPlanId: DEFAULT_PLAN_ID,
 };
 
+export const teamManagementProps = {
+  disableTeamMemberReassignment: DISABLE_TEAM_MEMBER_REASSIGNMENT,
```

Comment:
should the `opensrpBaseUrl` prop be added here too

Reply:
@p-netm that is passed by default to all PrivateComponent components